### PR TITLE
feat: Improve playground layout

### PR DESCRIPTION
**Pull request by tomayac**
Switch to CSS Grid

(Cherry-picked from #358)

### CODE REVIEW

**Review by alex-snezhko — APPROVED** [review]
LGTM, the playground is a lot more responsive now!

**Comment by spotandjake** [comment]
I like the look of this pr, two things I think the `console` is a little bit to long now, maybe `5/8`, `3/8` split? also I am having trouble with the compile button, something seems to be blocking my input but I couldn't find what.

**Comment by tomayac** [comment]
> I like the look of this pr, two things I think the `console` is a little bit to long now, maybe `5/8`, `3/8` split? also I am having trouble with the compile button, something seems to be blocking my input but I couldn't find what.

You can change the relations in line 46 and 65. I like the equal balance personally. Not sure about the other problem. I rescaled in all dimensions and multiple times, but the button always was reactive (I assume you did close the Netlify preview thing [twice]).

File: playground_src/stylesheets/playground.css [hunk 1]
```diff
@@ -40,34 +42,29 @@ body {
...
-    flex-grow: 1;
     white-space: pre-wrap;
 }
 
-@media only screen and (max-width: 768px) {
+@media only screen and (max-width: 1024px) {
     #editor-wrapper {
-        flex-direction: column;
+        grid-template-columns: 1fr;
+        grid-template-rows: 50% 50%;
```
Comment (tomayac) [inline playground_src/stylesheets/playground.css:65]: Change the vertical relations here if you want.

File: playground_src/stylesheets/playground.css [hunk 2]
```diff
@@ -40,34 +42,29 @@ body {
...
-    flex-direction: row;
-}
-
-#editor {
-    flex: 3;
-}
-.monaco-editor {
-    padding-top: 1rem;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
```
Comment (tomayac) [inline playground_src/stylesheets/playground.css:46]: Change the horizontal relations here if you want.

**Comment by ospencer** [comment]
Thanks @tomayac!

**Pull request merged**